src/pybind/mgr/zabbix/module.py

```diff
@@ -16,23 +16,24 @@ def avg(data):
 
 
 class ZabbixSender(object):
-    def __init__(self, sender, host, port, log):
+    def __init__(self, sender, host, port, hostname, log):
         self.sender = sender
         self.host = host
         self.port = port
+	self.hostname = hostname
```

Comment:
The indent in here is not right. So, can you fix it?

Reply:
ok